### PR TITLE
Oracle updates to Dockerfiles base image, added mandatory TPL and SECURITY.md files

### PR DESCRIPTION
**Pull request by desagar**

### CODE REVIEW

File: Dockerfile.dapper [hunk 1]
```diff
@@ -1,13 +1,26 @@
-FROM registry.suse.com/bci/golang:1.17
+# Copyright (C) 2020, 2021, Oracle and/or its affiliates.
```
Comment (fredtibbitts) [inline Dockerfile.dapper:1]: I know we have the outstanding question on whether the copyright should be added or not, but if we do end up adding the copyright it should be 2022.

File: package/Dockerfile [hunk 2]
```diff
@@ -1,27 +1,34 @@
-FROM registry.suse.com/bci/golang:1.17 AS helm
-RUN zypper -n install git
-RUN git -C / clone --branch release-v3.9.0 --depth=1 https://github.com/rancher/helm
-RUN make -C /helm
+# Copyright (C) 2020, 2021, Oracle and/or its affiliates.
```
Comment (fredtibbitts) [inline package/Dockerfile:1]: 2022 if the copyright stays.

File: package/Dockerfile [hunk 3]
```diff
@@ -1,27 +1,34 @@
...
-    zypper -n install bash-completion gzip jq tar unzip vim wget && \
-    zypper clean -a && rm -rf /tmp/* /var/tmp/* /usr/share/doc/packages/* /usr/share/doc/manual/* /var/log/*
+FROM ghcr.io/oracle/oraclelinux:7-slim
+
+RUN yum-config-manager --enable ol7_optional_latest && \
+    yum-config-manager --enable ol7_addons && \
+    yum update -y && \
+    yum install -y bash bash-completion gzip jq tar unzip vim wget && \
+    yum-config-manager --add-repo https://yum.oracle.com/repo/OracleLinux/OL7/olcne13/x86_64 && \
+    yum -y install kubectl-1.20.11-4.el7.x86_64 && \
```
Comment (fredtibbitts) [inline package/Dockerfile:27]: The upstream version installs kubectl 1.21.8, should this version match?

File: package/Dockerfile [hunk 4]
```diff
@@ -33,8 +40,8 @@ RUN echo 'shell:x:1000:1000:shell,,,:/home/shell:/bin/bash' > /etc/passwd && \
     mkdir /home/shell/.kube && \
     chown -R shell /home/shell && \
     chmod 700 /run
-COPY --from=helm ./helm/bin/helm /usr/local/bin/
-COPY --from=build /kubectl /k9s ./kustomize* /usr/local/bin/
+
+COPY --from=build /k9s ./kustomize* /usr/local/bin/
```
Comment (fredtibbitts) [inline package/Dockerfile:44]: I think we usually copy the security.md and TPL files too?

**Review by fredtibbitts — COMMENTED** [review]
A few minor comments.

File: package/Dockerfile (hunk 3 above)
Comment (desagar) [reply package/Dockerfile:27]: I did not find 1.21 in the yum repos we are using. Maybe there's another OL7 yum repo which has it. I will take a look

File: Dockerfile.dapper (hunk 1 above)
Comment (desagar) [reply Dockerfile.dapper]: done

File: package/Dockerfile (hunk 2 above)
Comment (desagar) [reply package/Dockerfile]: done

File: package/Dockerfile (hunk 3 above)
Comment (fredtibbitts) [reply package/Dockerfile:27]: Looks like 1.21 is available for OL7 in the OLCNE 1.4 repo: https://yum.oracle.com/repo/OracleLinux/OL7/olcne14/x86_64/index.html

Comment (desagar) [reply package/Dockerfile]: thanks, done

File: package/Dockerfile (hunk 4 above)
Comment (desagar) [reply package/Dockerfile:41]: done

**Review by fredtibbitts — APPROVED** [review]

**Pull request merged**